src/main/resources/db/migration/V1__Create_Team_Info_table.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE traffic.team_info(
+  id BIGSERIAL,
+  team_name VARCHAR(255) NOT NULL,
+  schedule_url VARCHAR(255) NOT NULL
+);
```

Comment:
There should be additional fields for each time including which type of league (major, minor, college"